caffe2/serialize/in_memory_adapter.h

```diff
@@ -2,12 +2,11 @@
 #include <caffe2/serialize/read_adapter_interface.h>
 #include <cstring>
 
-namespace caffe2 {
-namespace serialize {
+namespace caffe2::serialize {
 
 class MemoryReadAdapter final : public caffe2::serialize::ReadAdapterInterface {
  public:
-  explicit MemoryReadAdapter(const void* data, off_t size)
+  explicit MemoryReadAdapter(const void* data, size_t size)
```

Comment:
Why is this better?

Reply:
Because the negative read size has no sense, and size_t avoids some casts later.

Reply:
This is public API for caffe2 and you can see it being used in other places, so I don't think this kind of change is ok: https://github.com/search?q=%2FMemoryReadAdapter%2F+language%3AC%2B%2B&type=code&l=C%2B%2B
We should definitely always check github global search before doing this kind of BC-breaking changes!

Reply:
Thank you for pointing it out. The change has been reverted.